src/command_modules/azure-cli-sql/azure/cli/command_modules/sql/_params.py

```diff
@@ -124,653 +126,602 @@ def _configure_db_create_params(
-
-
-#####
-#           sql server key
-#####
-
-
-with ParametersContext(command='sql server key') as c:
-    c.argument('server_name', arg_type=server_param_type)
-    c.argument('key_name', options_list=('--name', '-n'))
-    c.argument('kid',
-               options_list=('--kid', '-k'),
-               required=True,
-               help='The Azure Key Vault key identifier of the server key. An example key identifier is '
-               '"https://YourVaultName.vault.azure.net/keys/YourKeyName/01234567890123456789012345678901"')
-
-
-#####
-#           sql server tde-key
-#####
-
-
-with ParametersContext(command='sql server tde-key') as c:
-    c.argument('server_name', arg_type=server_param_type)
-
-with ParametersContext(command='sql server tde-key set') as c:
-    c.argument('kid',
-               options_list=('--kid', '-k'),
-               help='The Azure Key Vault key identifier of the server key to be made encryption protector.'
-               'An example key identifier is '
-               '"https://YourVaultName.vault.azure.net/keys/YourKeyName/01234567890123456789012345678901"')
-    c.argument('server_key_type',
-               options_list=('--server-key-type', '-t'),
-               help='The type of the server key',
-               **enum_choice_list(ServerKeyType))
-
-
-#####
-#           sql server vnet-rule
-#####
-
-
-with ParametersContext(command='sql server vnet-rule') as c:
-    # Help text needs to be specified because 'sql server vnet-rule create' is a custom
-    # command.
-    c.argument('server_name',
-               options_list=('--server', '-s'), completer=get_resource_name_completion_list('Microsoft.SQL/servers'))
-
-    c.argument('virtual_network_rule_name',
-               options_list=('--name', '-n'))
-
-    c.argument('virtual_network_subnet_id',
-               options_list=('--subnet'),
-               help='Name or ID of the subnet that allows access to an Azure Sql Server. '
-               'If subnet name is provided, --vnet-name must be provided.')
-
-    c.argument('ignore_missing_vnet_service_endpoint',
-               options_list=('--ignore-missing-endpoint', '-i'),
-               help='Create firewall rule before the virtual network has vnet service endpoint enabled',
-               **three_state_flag())
-
-register_extra_cli_argument('sql server vnet-rule create', 'vnet_name', options_list=('--vnet-name'),
-                            help='The virtual network name', validator=validate_subnet)
+        arg_ctx.ignore('recovery_services_recovery_point_resource_id')
+
+
+# pylint: disable=too-many-statements
+def load_arguments(self, _):
+
+    with self.argument_context('sql') as c:
+        c.argument('location_name', arg_type=get_location_type(self.cli_ctx))
+        c.argument('usage_name', options_list=('--usage', '-u'))
+
+    with self.argument_context('sql db') as c:
+        c.argument('database_name',
+                   options_list=('--name', '-n'),
```

Comment:
Can you run your magic regex to replace the tuples with lists?

Reply:
Done. I just forgot to do it